manifests/ccm.yaml

```diff
@@ -69,11 +69,11 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: cloud-controller-manager
+  name: k3s-cloud-controller-manager
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cloud-controller-manager
+  name: k3s-cloud-controller-manager
 subjects:
 - kind: User
   name: cloud-controller-manager
```

Comment:
This needs to be changed too, along with updating the cloud-controller-manager certificate subject here:
https://github.com/k3s-io/k3s/blob/master/pkg/daemons/control/deps/deps.go#L297

Reply:
Changed in new commit.

Reply:
https://github.com/k3s-io/k3s/pull/3388#pullrequestreview-674630108, this should fix this problem

Reply:
The manifest can have `k3s` embedded in it, since RKE2 does not use any of the package manifests (they are disabled via build flag).